run_tests.py

```diff
@@ -501,10 +501,11 @@ def run_test(testname, host, target):
                              match, width, exe_name)
                     exe_name = "./" + exe_name
                     cc_cmd += " -DTEST_ZEBIN" if options.ispc_output == "ze" else " -DTEST_SPV"
-
             ispc_cmd = ispc_exe_rel + " --woff %s -o %s --arch=%s --target=%s -DTEST_SIG=%d" % \
                         (filename, obj_name, options.arch, xe_target if target.is_xe() else options.target, match)
-
+            # Produce position independent code. Required when host compiler is clang 15.0+.
+            if host.is_linux():
+                ispc_cmd += " --pic"
```

Comment:
Clang 15 changed default from `-mrelocation-model static` to `-mrelocation-model pic`. I think if we are going to use PIC consistently, we should add it to both c++ and ispc compilations, so it's PIC across the board (regardless compiler or compiler versions).